### PR TITLE
Gradle 8

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -8,5 +8,6 @@ gradleEnterprise {
     buildScan {
         termsOfServiceUrl = 'https://gradle.com/terms-of-service'
         termsOfServiceAgree = 'yes'
+        publishAlways()
```
Comment (melix) [inline settings.gradle:11]: Can you remove this? We don't publish always because we cannot accept the terms of service on behalf of a contributor

Comment (Goooler) [reply settings.gradle:11]: How about `publishAlwaysIf` on CI?

Comment (melix) [reply settings.gradle:11]: Yes, that we can do

Comment (Goooler) [reply settings.gradle]: e34b9be69c2f586ed038193558c9740e3c71f6ca

**Comment by melix** [comment]
Thanks for your contribution again!

**Pull request merged**